itest/src/edu/stanford/nlp/dcoref/DcorefExactOutputITest.java

```diff
@@ -4,10 +4,11 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.Map;
+import java.util.Map.Entry;
```

Comment:
Please keep functional changes (the Map.Entry change, for example) separate from syntactic sugar changes, which as I've mentioned elsewhere, we really don't want to spend effort on